runtimes/aleph-debian-11-python/init1.py

```diff
@@ -325,6 +325,23 @@ async def make_request(session, scope):
     return headers, body
 
 
+def show_loading():
+    with open("/root/loading.html", "r") as f:
+        body = {
+            "body": f.read()
```

Comment:
Since the entire body is loaded in memory, would it not be easier to skip the file descriptor ?

```suggestion
            "body": Path(""/root/loading.html").read_bytes()
```

Reply:
As I commented in the previous comment, I think that is better to use string format instead bytes, but yes, if you want I can skip the file descriptor by `Path(""/root/loading.html").read_text()`, but I don't see the advantage.

Reply:
Simpler code is better. The overhead created by using a manager would be avoided.

Reply:
Okay, I just changed it